src/test/kotlin/de/jodamob/kotlin/testrunner/sample/TestWithRunner.kt

```diff
@@ -1,15 +1,17 @@
 package de.jodamob.kotlin.testrunner.sample
 
 import de.jodamob.kotlin.testrunner.KotlinTestRunner
+import de.jodamob.kotlin.testrunner.OpenedClasses
+import de.jodamob.kotlin.testrunner.nonopen.FinalNonOpenClassSample
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito
 import org.mockito.exceptions.base.MockitoException
 
 @RunWith(KotlinTestRunner::class)
+@OpenedClasses("de.jodamob.kotlin.testrunner.sample")
```

Comment:
yes I saw that but it would be more obvious if the name would contain package, dont you think?
We could even simply extend the class one and don't do everything separate